### PR TITLE
Add vcpkg usage in README.md

### PR DESCRIPTION
**Pull request by eclipse0922**
add vcpkg usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,17 +84,31 @@ Detailed Doxygen-genereated documentation is [available](https://codedocs.xyz/ey
...
+  include(FetchContent)
+  FetchContent_Declare(cuda-api-wrappers_library
+      GIT_REPOSITORY https://github.com/eyalroz/cuda-api-wrappers.git
+      GIT_TAG v12.34.56 # Replace this with a real available version
+  )
+  FetchContent_MakeAvailable(cuda-api-wrappers_library)
+  ```
+***Incorporating via vcpkg***
+
+* Install cuda-api-wrappers using [vcpkg](https://github.com/microsoft/vcpkg) by running:
```
Comment (eyalroz) [inline README.md:108]: If I use vcpkg to install the library, do I then just write `find_package(cuda-api-wrappers)`?

File: README.md [hunk 2]
```diff
@@ -84,17 +84,31 @@ Detailed Doxygen-genereated documentation is [available](https://codedocs.xyz/ey
...
-   include(FetchContent)
-   FetchContent_Declare(cuda-api-wrappers_library
-       GIT_REPOSITORY https://github.com/eyalroz/cuda-api-wrappers.git
-       GIT_TAG v12.34.56 # Replace this with a real available version
-   )
-   FetchContent_MakeAvailable(cuda-api-wrappers_library)
-   ```
-   The _same_ target names, with the namespaces, will be available in this case.
+***General CMAKE Usage***
+* After incorporating the library via either FetchContent or vcpkg, use find_package(cuda-api-wrappers) in your CMake project. Make sure the library's install location or build directory is in CMake's package search path.
```
Comment (eyalroz) [inline README.md:88]: That's not exactly what I want to say... but I'll amend the REAMDE.md to mention the vcpkg availability.

Comment (eclipse0922) [reply README.md:88]: My mistake, I modified too much.
This PR is just a suggestion, 
If you have questions about vcpkg, I can help you as far as I know.

**Comment by eyalroz** [comment]
I've changed README.md somewhat on the development branch, let me know what you think.

**Comment by eclipse0922** [comment]
@eyalroz Looks good to me. 👍

**Comment by eyalroz** [comment]
By the way, if you have a suggestion about how to write the meta-data for putting a library onto the _conan_ package management system, I'd be glad to get a reference, since I tried figuring that out last year and got lost.

**Comment by eclipse0922** [comment]
Unfortunately, I haven't tried conan pacakger yet. 
I know conan is also used a lot, so it would be nice to learn how to use/write conan when the time comes, but I haven't done it yet. 
I'll try when I have time.

**Pull request closed without merging**